src/electron/views/left-nav/fluent-left-nav.scss

```diff
@@ -13,6 +13,14 @@
             padding: 0px;
         }
     }
+    .nav-menu-container {
+        display: flex;
+        min-height: $detailsViewCommandBarHeight;
```

Comment:
Definitely think we should have a PR to rename this style and a few others to more generic names.